### DISCUSSION TITLE
Notifications and emails only send once and only once

### DISCUSSION THREAD

**Discussion by andyaws** [inventree/InvenTree#6141] [General]
Have an issue where inventree only sends a notification to the web ui once per stock item, and only ever sends one email.
if i deplete stock on a new item i get the email and notification in the web ui.
after which i dont get any more notificaitons, e.g if i add more stock then deplete, i dont get any further notificaitons.

**Comment by matmair** [comment]
There is a one day back off for each notification type for mails by default

**Reply by andyaws** [reply]
aah that would explain it.
will test 24hours from now.

cheers

**Reply by andyaws** [reply]
what about notifications in the web ui?

**Reply by matmair** [reply]
Docs around notification stacks are pretty barebones so reasonable that you had to ask

**Comment by matmair** [comment]
> what about notifications in the web ui?

could be a bug; @andyaws  would you mind filling out a bug report so I can try replicating tomorrow?

**Reply by andyaws** [reply]
sure where do you want it sending to?

**Reply by matmair** [reply]
On this repo please, we have a [bug report template](https://github.com/inventree/InvenTree/issues/new?assignees=&labels=bug%2Cquestion%2Ctriage%3Anot-checked&projects=&template=bug_report.yaml) for issues.
